### PR TITLE
Change Order Feedback System to Thumbs Up/Down

### PR DESCRIPTION
**Pull request by Brandons42**
Changed UI for order feedback submission from star rating to thumbs up/down and changed user flow for issue submission according to Figma designs. Updated logic to interact with server appropriately given the new feedback system. Fixed bugs from previous system.

Should be pushed with associated pull request from attain-server, https://github.com/josephycyeh/attain-server/pull/28

### CODE REVIEW

**Comment by Brandons42** [comment]
https://user-images.githubusercontent.com/20510288/196289259-a4063862-8fdf-4de7-8e87-6d8aae83ae45.MP4

Positive feedback flow

File: screens/orderSubmittedScreen/orderSubmittedScreen2.js [hunk 1]
```diff
@@ -191,35 +215,55 @@ export default function OrderSubmittedScreen2({ navigation, route }) {
                 <Modal
                     animationType="slide"
                     transparent={true}
-                    visible={modalVisible}
+                    visible={thumbsModalVisible}
                     onRequestClose={() => {
                     Alert.alert("Modal has been closed.");
```
Comment (josephycyeh) [inline screens/orderSubmittedScreen/orderSubmittedScreen2.js:220]: Is this logging or does it actually display an alert

**Comment by Brandons42** [comment]
https://user-images.githubusercontent.com/20510288/196289427-4f3eee00-01a7-45c8-9748-1334d1171ff9.MP4

Negative feedback flow

File: screens/orderSubmittedScreen/orderSubmittedScreen2.js [hunk 2]
```diff
@@ -191,35 +215,55 @@ export default function OrderSubmittedScreen2({ navigation, route }) {
...
+                        </View>
                     </View>
                     </View>
                 </Modal>
+                <Modal
+                    animationType="slide"
+                    transparent={true}
+                    visible={issuesModalVisible}
+                    onRequestClose={() => {
+                    Alert.alert("Modal has been closed.");
```
Comment (josephycyeh) [inline screens/orderSubmittedScreen/orderSubmittedScreen2.js:249]: same here

File: screens/orderSubmittedScreen/orderSubmittedScreen2.js (hunk 1 above)
Comment (Brandons42) [reply screens/orderSubmittedScreen/orderSubmittedScreen2.js]: Resolved in new commit.

File: screens/orderSubmittedScreen/orderSubmittedScreen2.js (hunk 2 above)
Comment (Brandons42) [reply screens/orderSubmittedScreen/orderSubmittedScreen2.js]: Resolved in new commit.

**Pull request merged**